lib/display.rb

```diff
@@ -0,0 +1,23 @@
+class Display
+
+  attr_reader :board
+  attr_accessor :display_array
+
+  def initialize(board)
```

Comment:
Why pass in the board when you're going to use a hardcoded array? Is this just a placeholder value?

Reply:
That board was part of an earlier version of Display that didn't get used and should've been deleted. It's irrelevant to the Display now. DELETED.